cip-builtin-drop/README.md

```diff
@@ -0,0 +1,140 @@
+---
+CIP: ?
+Title: New Plutus Builtin dropList
+Status: Proposed
+Category: Plutus
+Authors:
+    - Philip DiSarro <info@anastasialabs.com>
+Implementors: []
+Discussions: 
+    - https://github.com/cardano-foundation/CIPs/pull/418/
```

Comment:
```suggestion
    - https://github.com/cardano-foundation/CIPs/pull/418
    - https://github.com/cardano-foundation/CIPs/pull/767
```
I presume that the first of these links represents one of the "design patterns" for which this would be useful (please correct if the older link is unrelated).